src/components/header/PatientHeader.jsx

```diff
@@ -1,22 +1,96 @@
 import React, { PureComponent } from 'react';
+import { connect } from "react-redux";
 import PropTypes from 'prop-types';
 import dateFns from 'date-fns';
 import { Glyphicon } from 'react-bootstrap';
 import { withRouter } from "react-router";
+import { selectors } from '../../store';
 import { DATE_FORMAT } from "../../constants";
 import patientUtil from '../../domain/patient/patientUtil';
 import '../../../assets/css/patientHeader.css';
 
 export class PatientHeader extends PureComponent {
-
   constructor(props) {
     super(props);
     // converts from REST rep but *only if necessary*; handles already-converted Patient object as well
     this.state = {
-      patient: patientUtil.createFromRestRep(this.props.patient)
+      patient: patientUtil.createFromRestRep(this.props.patient),
+      patientIdentifiers: [],
+      additionalPatientIdentifiers: [],
+      shouldDisplayAdditionalPatientIdentifier: false
     };
+
+    this.togglePatientIdentifierDisplay = this.togglePatientIdentifierDisplay.bind(this);
   };
 
+  componentDidMount() {
+    if (this.state.patient) {
```

Comment:
@jomadoye all the special logic that deals with CCC and HCC, etc, needs to live in the APZU-IC3... that's why I was hoping all this functionality could live in a callback function specified in APZU-IC3 and passed in as a prop "identifiersToDisplay".  Is there a reason this wouldn't be possible?